### PR TITLE
fix(mainHeader): `touch target`s were reduced to the `text` instead o…

### PR DESCRIPTION
**Pull request by marcel-bitfly**
…f the `line`

See: BEDS-346

### CODE REVIEW

File: frontend/components/bc/header/MainHeader.vue [hunk 1]
```diff
@@ -68,20 +68,25 @@ const toggleMegaMenu = (evt: Event) => {
...
+const userMenu: UserMenuItem[] = [
+  {
+    label: labelSettings,
+  },
+  {
+    label: labelLogout,
+  },
+]
+const handleUserMenuSelect = async (value: UserMenuItem) => {
+  if (value.label === labelSettings) {
```
Comment (MauserBitfly) [inline frontend/components/bc/header/MainHeader.vue:83]: I am not sure if this solution is salable if we have more options and also using the translation as Identifier is not the best choice I think.  

Here is a suggestions:
```suggestion
type UserMenuItem = { command?: () => (Promise<void> | void), label: string }
const userMenu: UserMenuItem[] = [
  {
    command: async () => { await navigateTo('/user/settings') },
    label: $t('header.settings'),
  },
  {
    command: () => doLogout(),
    label: $t('header.logout'),
  },
]
const handleUserMenuSelect = async (value: UserMenuItem) => {
  await value.command?.()
}
```

**Review by MauserBitfly — COMMENTED** [review]
Works fine, but I left a suggestion for better scalability.

File: frontend/components/bc/header/MainHeader.vue (hunk 1 above)
Comment (marcel-bitfly) [reply frontend/components/bc/header/MainHeader.vue]: I am following yr suggestion...thx

**Review by MauserBitfly — APPROVED** [review]
LGTM

**Pull request merged**